examples/website-demos/virtual-event/src/components/CentralPanel/EventCard.scss

```diff
@@ -0,0 +1,84 @@
+@import '../../styles/variables';
+
+.event-card {
+  &-container {
+    display: flex;
+    overflow: hidden;
+    box-shadow: 0px 0px 8px rgba(0, 0, 0, 0.15);
+    border-radius: 16px;
+    margin: 32px;
+    height: 156px;
+  }
+
+  &-image {
+    width: 250px;
+  }
+
+  &-content {
+    display: flex;
+    margin: 16px;
+    flex-direction: column;
+    width: 100%;
+
+    &-content {
+      display: flex;
+      flex: 1;
+      font-size: 14px;
+    }
+  }
+
+  &-title {
+    display: flex;
+    flex: 1;
+
+    &-text {
+      font-weight: bold;
+      margin: 0 8px;
+    }
+  }
+
+  &-footer {
+    display: flex;
+    flex: 1;
+    justify-content: space-between;
+  }
+
+  &-viewers {
+    display: flex;
+
+    &-left {
+      display: flex;
+      align-items: center;
+
+      &-count {
+        margin-left: 8px;
+      }
+    }
+
+    &-right {
+      display: flex;
+      align-items: center;
+      margin-left: 16px;
+      &-count {
+        margin-left: 8px;
+      }
+    }
+  }
+
+  &-label {
+    display: flex;
+    align-items: center;
+    padding: 4px 8px;
+    background: $light-grey;
+    border-radius: 999px;
+    height: fit-content;
+
+    &.moderated {
+      background: #fbca6a;
```

Comment:
does this need to be a variable?

Reply:
For all colors I didnt find in variable list I left static

Reply:
does it have a static variable? if so add it, in case they ever change